crates/sui/src/sui_commands.rs

```diff
@@ -59,35 +59,43 @@ const CONCURRENCY_LIMIT: usize = 30;
 const DEFAULT_EPOCH_DURATION_MS: u64 = 60_000;
 const DEFAULT_FAUCET_NUM_COINS: usize = 5; // 5 coins per request was the default in sui-test-validator
 const DEFAULT_FAUCET_MIST_AMOUNT: u64 = 200_000_000_000; // 200 SUI
+const DEFAULT_FAUCET_PORT: u16 = 9123;
+#[cfg(feature = "indexer")]
+const DEFAULT_GRAPHQL_PORT: u16 = 9125;
+#[cfg(feature = "indexer")]
+const DEFAULT_INDEXER_PORT: u16 = 9124;
 
 #[cfg(feature = "indexer")]
 #[derive(Args)]
 pub struct IndexerFeatureArgs {
-    /// Start an indexer with default host and port: 0.0.0.0:9124, or on the port provided.
+    /// Start an indexer with default host and port: 127.0.0.1:9124. This flag accepts also a port,
+    /// a host, or both (e.g., 127.0.0.1:9124).
     /// When providing a specific value, please use the = sign between the flag and value:
-    /// `--with-indexer=6124`.
+    /// `--with-indexer=6124` or `--with-indexer=0.0.0.0`, or `--with-indexer=0.0.0.0:9124`
     /// The indexer will be started in writer mode and reader mode.
     #[clap(long,
-            default_missing_value = "9124",
+            default_value = "127.0.0.1:9124",
```

Comment:
You should now no longer need to call `unwrap_or_default` on this value -- if it's `None`, then we are not going to run the indexer at all -- so you can remove this once again.

Reply:
We decided that `with-graphql` implies `--with-indexer`. If we keep that, then this is needed, unless we want to manually construct the IP + port instead of unwrapping a default.